style.css

```diff
@@ -0,0 +1,242 @@
+body {
+  box-sizing: border-box;
+  font-size: 18px;
+  background-color: #191765;
+  margin: 0;
+  color: white;
+  z-index: -1;
+  font-family: 'Roboto', sans-serif;
+}
+
+/* HEADER OR TOOLBAR */
+
+figure a {
+  text-decoration: none;
+  color: #fff;
+  font-weight: 500;
+  text-align: center;
+  font-style: normal;
+  padding: 10px;
+  position: absolute;
+}
+
+nav li a {
+  text-decoration: none;
+  font-style: normal;
+  display: flex;
+  font-weight: 500px;
+  color: #fff;
+  align-items: center;
+  text-align: center;
+  padding: 10px;
+}
+
+nav ul li {
+  list-style-type: none;
+}
+
+ul {
+  flex: 1;
+  display: flex;
+}
+
+nav {
+  display: flex;
+}
```

Comment:
This section looks very dissimilar to the template design.

<img width="190" alt="Screen Shot 2022-08-19 at 7 45 38" src="https://user-images.githubusercontent.com/82426682/185621355-0bccf178-fa4d-4cef-8995-84c52fec7660.png">

<img width="341" alt="Screen Shot 2022-08-19 at 7 45 56" src="https://user-images.githubusercontent.com/82426682/185621410-3a9c8f1d-5a36-4bfd-8798-aad9141f00d0.png">

Please fix it.

<img width="487" alt="Screen Shot 2022-08-19 at 7 46 19" src="https://user-images.githubusercontent.com/82426682/185621484-f800c06e-394f-4d69-ab3d-8d5aed845e64.png">